app/src/main/java/io/intrepid/contest/screens/contestcreation/categorieslist/CategoriesListFragment.java

```diff
@@ -98,4 +98,16 @@ public Category getDefaultCategory(int categoryNameRes, int categoryDescriptionR
         String categoryDescription = getString(categoryDescriptionRes);
         return new Category(categoryName, categoryDescription);
     }
+
+    @Override
+    public void setNextEnabled(boolean enabled) {
+        ((EditContestContract) getActivity()).setNextEnabled(enabled);
+    }
+
+    @Override
+    public void onFocus() {
+        if (presenter != null) {
```

Comment:
Yeah, this lifecycle stuff is a little obscure.
You can add a comment saying that if the presenter is null it's expected that the lifecycle will call `presenter.onViewBound()` anyway